### PR TITLE
docs: Allow react-native components to run in Storybook Web

### PR DESCRIPTION
**Pull request by darryltec**
## Motivations

- This PR allows us to run React native components in web.
- This doesn't add any examples for it, just dependencies

### Added

- Install necessary packages to run react native in web storybook

### Changed

- adjusted main.js to know how to run RN in web
- Adjusted divider example for web so it shows the vertical example cleanly

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

```diff
diff --git a/packages/components/src/Divider/Divider.stories.mdx b/packages/components/src/Divider/Divider.stories.mdx
index 581a49e9..0a132954 100644
--- a/packages/components/src/Divider/Divider.stories.mdx
+++ b/packages/components/src/Divider/Divider.stories.mdx
@@ -1,4 +1,5 @@
 import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Divider as MobileDivider } from "@jobber/components-native";
 import { Content } from "@jobber/components/Content";
 import { Text } from "@jobber/components/Text";
 import { Heading } from "@jobber/components/Heading";
@@ -28,6 +29,7 @@ import { Divider } from "@jobber/components/Divider";
       >
         <Content>Some amazing content</Content>
         <Divider {...args} />
+        <MobileDivider {...args} />
         <Content>Even more amazing content</Content>
       </div>
     )}
```

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -122,7 +127,8 @@
     "ts-node": "^10.9.1",
     "typed-css-modules": "^0.7.0",
     "typed-css-modules-loader": "0.0.18",
-    "typescript": "^4.9.5"
+    "typescript": "^4.9.5",
+    "webpack": "^4.46.0"
```
Comment (rodrigoeidelvein) [inline package.json:131]: Do we need this installed?

I uninstalled it and it still worked

**Review by rodrigoeidelvein — COMMENTED** [review]
Just a question around webpack

File: package.json (hunk 1 above)
Comment (darryltec) [reply package.json:131]: hmmm... you's right!

**Review by rodrigoeidelvein — APPROVED** [review]
LGTM! Also loved the fix to the Divider direction!

**Pull request merged**